### ISSUE TITLE
remove unused dependencies from @ngneat/transloco package

### ISSUE THREAD

**Issue by FabienDehopre** [ngneat/transloco#692]
The @ngneat/transloco package have several unused dependencies that may cause conflicts with the same dependencies in different version:
- fs-extra
- glob
- replace-in-file
- ora
- lodash.kebabcase
- @ngneat/transloco-utils

I propose to remove them to reduce the package install size and any conflicts they may cause when install @ngneat/transloco package in project/workspace with complex dependencies installed.

I will provide a PR to remove the unused dependencies from the package.json (an ng-packagr.json).

**Comment by shaharkazaz** [comment]
@FabienDehopre These dependecies are used by the schematics, they need to be there

**Comment by FabienDehopre** [comment]
@shaharkazaz ok... but then, why does the schematics live in a separated package (transloco-schematics)? Can't they be deployed/released in separated package (as @ngneat/transloco-schematics)? 

I created this issue and the related PR because the version 5.x of transloco completely breaks my workspace because of the `glob` dependency which, in transloco, is >= 10.0.0. In my work space, I had to pin it to 9.3.5 because version 10 broke the compatibility with nx (at least on my workspace) for some unknown reason.

**Comment by FabienDehopre** [comment]
@shaharkazaz another possibility would be to move those dependencies to `peerDependencies` and mark them as optional. So, those who need to use the schematics should also install those optional peer dependencies.

**Comment by shaharkazaz** [comment]
They are in a different lib for development convenience. 
I need to check if releasing them in a different package is doable.

Adding them as optional peers is not an option from my end as this will result in a bad DX IMO, this should be seamless to anyone using Transloco.

Figuring out why glob isn't working as expected on your end is also a solution 🙂  I've worked with several NX workspaces but didn't encounter any issue related to glob.

I think you can close the PR as removing these dependencies won't be the solution, I need to think about how I want to handle this.

**Comment by FabienDehopre** [comment]
@shaharkazaz I finally succeeded to solve my issue by adding a resolution for the warp-ansi package.

**Comment by shaharkazaz** [comment]
Glad to hear!

**Comment by Celtian** [comment]
Thanks a lot.... This helped:

```json
{
  "resolutions": {
    "wrap-ansi": "^7.0.0"
  }
}
```

**Comment by FabienDehopre** [comment]
This is just sad that we now need to add a resolution for the wrap-ansi package in all projects using transloco v5 :'(

**Comment by shaharkazaz** [comment]
@FabienDehopre can you provide information on why you need to add it?
Which library is using the outdated version?
I think that generally I can set the dependency to a lower version, but let's understand the root cause and not just be sad about it :)

**Comment by Celtian** [comment]
Probably `cliui` ...

**Comment by FabienDehopre** [comment]
Indeed, it's cliui which still depends on version ^7.0.0 of wrap-ansi which is not ESM compatible.
If I'm not mistaken, there is a fork of cliui called @isaacs/cliui which depends on wrap-ansi version ^8.0.0 which is ESM compatible.
